.github/workflows/test_pr.yml

```diff
@@ -34,4 +34,38 @@ jobs:
         with:
           args: deploy -P roster-pr --only hosting:project-dashboard-pull-request
         env:
-          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
\ No newline at end of file
+          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}   
+                    
+      # Only run the coverage once
+      - name: get value percentage and print
+      
+        run: |
+          # var REF = 'refs/pull/27/merge.json';
+          REF=${{ github.ref }}
+          # console.log('github.ref: ' + REF);
+          echo "github.ref: $REF"
+          # var PATHS = REF.split('/');
+          IFS='/' read -ra PATHS <<< "$REF"
+          # var COVERAGEBADGE = PATHS[1] + PATHS[2];
+          COVERAGEBADGE="${PATHS[1]}_${PATHS[2]}"
+          # console.log(COVERAGEBADGE); // 'pull_27'
+          echo $COVERAGEBADGE
+          # process.env.COVERAGEBADGE = 'pull_27';
+          echo "COVERAGEBADGE=$(echo ${COVERAGEBADGE})" >> $GITHUB_ENV
+          PERCENTAGE=$(npm test -- --coverage --coverageReporter='text-summary' | grep 'All files' | cut -d '|' -f 5)
+          echo $PERCENTAGE
+          echo "COVERAGE=$PERCENTAGE" >> $GITHUB_ENV
```

Comment:
Looks like this did not work. Try echoing the variable like above when echoing the branch.

https://gist.github.com/RonanCadot/97cd1718d218b973104e73446b5cdeec